actor/v2action/service_instance_test.go

```diff
@@ -26,6 +26,285 @@ var _ = Describe("Service Instance Actions", func() {
 		actor = NewActor(fakeCloudControllerClient, nil, nil)
 	})
 
+	Describe("CreateServiceInstance", func() {
+		var (
+			serviceInstance       ServiceInstance
+			createServiceWarnings Warnings
+			createServiceErr      error
+			returnedInstance      ccv2.ServiceInstance
```

Comment:
It looks like `returnedInstance` is only initialized in an inner `When`, which creates an opportunity for state to leak between tests. For example, https://github.com/cloudfoundry/cli/pull/1526/files#diff-be7aaf8df0722122e8627ba39f0dd8c1R297 may run when this variable is or is not initialized. Would you mind either moving `returnedInstance`'s declaration to a narrower scope, since it's not really being used on 297, or initializing it explicitly in a BeforeEach at the scope where it's declared?

Reply:
makes sense, we've fixed it , thanks